src/System.Runtime.Extensions/src/System/Diagnostics/Stopwatch.Windows.cs

```diff
@@ -2,18 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.CompilerServices;
+
 namespace System.Diagnostics
 {
     public partial class Stopwatch
     {
-        private static bool QueryPerformanceFrequency(out long value)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why do we need this? This is about as simple a call as can be... if it's not getting inlined but should, shouldn't that be addressed in the JIT?

Reply:
I didn't actually check if this was or was not inlining. I've just always marked simple methods like this as `AggressiveInlining` to help ensure that they are in fact inlined.

Reply:
Let's trust in the JIT, and if it does the "wrong" thing, fix it.
cc: @AndyAyersMS

Reply:
`AggressiveInlining` should not be needed for methods that just wrap other methods like this.

Reply:
My rule is that every use of `AggressiveInlining` should come with numbers that demonstrate why it is required.

Reply:
Fixed.